README.md

```diff
@@ -180,6 +180,32 @@ windows:
 '------------------'------------------'
 ```
 
+### Two pane window with focus in second pane
+
+```yaml
+windows:
+  - name: sample-two-panes
+    root: ~/Code/sample/www
+    layout: even-horizontal
+    panes:
+      - rails server
+      - { commands: [ 'rails console' ], focus: true }
```

Comment:
Agreed, but it doesn't work.

What have I misunderstood?  (This is why I opened #88 in fact ; I didn't understand the required syntax.)

Given:

```yaml
windows:
  - name: "rails: s/c"
    root: "~/Documents/Personal/Source/witness-the-fitness/rails"
    panes:
      - bundle exec rails s
      - commands: bundle exec rails c
        focus: true
```

I get error:

```
/Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/pane.rb:7:in `block in as_tmux': undefined method `join' for "bundle exec rails c":String (NoMethodError)
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/pane.rb:5:in `tap'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/pane.rb:5:in `as_tmux'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/window.rb:41:in `map'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/window.rb:41:in `block in as_tmux'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/window.rb:26:in `tap'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/window.rb:26:in `as_tmux'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/session.rb:21:in `map'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/session.rb:21:in `block in as_tmux'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/session.rb:19:in `tap'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/tmux/session.rb:19:in `as_tmux'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/layout.rb:37:in `shell_commands'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/layout.rb:7:in `execute!'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/lib/teamocil/cli.rb:21:in `run!'
        from /Users/rmichael/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/teamocil-1.1/bin/teamocil:7:in `<top (required)>'
        from /Users/rmichael/.rbenv/versions/2.1.5/bin/teamocil:23:in `load'
        from /Users/rmichael/.rbenv/versions/2.1.5/bin/teamocil:23:in `<main>'
```

Quoting `command: 'bundle exec rails c'` also fails.

However, as commented in #88, an array works ; I'll update to use that form.  (It is indeed nicer than the explicit Hash.)

```
- commands: [ 'bundle exec rails c' ]
  focus: true
``